config/application.rb

```diff
@@ -64,16 +64,7 @@ def config.database_configuration
     # Allows us to skip minifincation on some files
     config.assets.skip_minification = []
 
-    # explicitly precompile any images in plugins ( /assets/images ) path
-    config.assets.precompile += [lambda do |filename, path|
-      path =~ /assets\/images/ && !%w(.js .css).include?(File.extname(filename))
```

Comment:
by moving this to manifest we lost some of the explicitness here ... this is only looking at non js/css files